lista-de-exercicios-java/src/main/java/org/example/exercicio13/Aluno.java

```diff
@@ -0,0 +1,53 @@
+package org.example.exercicio13;
+
+import java.util.ArrayList;
+
+public class Aluno {
+    private double nota;
+    private ArrayList<Double> notas;
+    private double media;
+    private String status;
+
+    public Aluno() {
+        this.notas = new ArrayList<>();
+    }
+
+    public void adicionarNota(double nota) {
+        if (nota < 0 || nota > 10) {
+            throw new IllegalArgumentException("A nota deve estar entre 0 e 10.");
+        }
+        this.notas.add(nota);
+    }
+
+    public ArrayList<Double> getNotas() {
+        return notas;
+    }
+
+    public double calcularMedia() {
+        double soma = 0;
+        for (int i = 0; i < this.notas.size(); i ++) {
```

Comment:
Apenas uma dica, da uma olhada nos outros tipos de for que é possível fazer, o código fica bem mais simples e é mais fácil de fazer, neste caso, por exemplo, poderia substituir por for(nota : notas){logica}, ou então forEach(nota -> {logica})